### ISSUE TITLE
cannot find module

### ISSUE THREAD

**Issue by limaologic** [joeferner/node-java#29]
I use the example lucene
but it throws an error:
limaoubuntu@limaoubuntu-VirtualBox:~/node_modules/java/examples/lucene$ node example.js 

node.js:201
        throw e; // process.nextTick error, or 'error' event on first tick
              ^
Error: Cannot find module '/home/limaoubuntu/node_modules/java/build/Release/nodejavabridge_bindings.node'
    at Function._resolveFilename (module.js:332:11)
    at Function._load (module.js:279:25)
    at Module.require (module.js:354:17)
    at require (module.js:370:17)
    at Object.<anonymous> (/home/limaoubuntu/node_modules/java/lib/nodeJavaBridge.js:5:16)
    at Module._compile (module.js:441:26)
    at Object..js (module.js:459:10)
    at Module.load (module.js:348:32)
    at Function._load (module.js:308:12)
    at Module.require (module.js:354:17)

**Comment by erizoy** [comment]
When I build my project with electron-packager, on Windows Server 2008 x64, I'm getting working application, but when I'm trying to run this app on another machine (Windows 7 x64), I get error:
![error](https://cloud.githubusercontent.com/assets/1070299/15887878/519b90fa-2d6d-11e6-86b0-5d02a61c4755.PNG)
What can I do wrong?
JAVA_HOME is setup correctly on both machines.

**Comment by antoinepairet** [comment]
@erizoy Did you find a solution?
I am having the same issue when I try to distribute an electron application using node-java

**Comment by coco-napky** [comment]
Any update on this?

**Comment by jmrama** [comment]
One hint that could be the reason of this problem.
After building the project with electron-packager, when I search under "node_modules\java" folder I've found more then 500 references with the absolute path to the folder where I have the program on my computer.
I've installed some global libraries to run this project but on the java.node under "node_modules\java\bin\win32-x64-53" there’s configured the same absolute path to the file "nodejavabridge_bindings.pdb".

Can someone verify if this effect is also happening on your project?
Thanks.
